src/test/java/com/bytezone/dm3270/TerminalClientTest.java

```diff
@@ -115,7 +130,10 @@ public void teardown() throws Exception {
     client.disconnect();
     service.stop(TIMEOUT_MILLIS);
   }
-
+  @Mock
+  private Screen screenMock;
+  private ScreenContext screenContextMock;
```

Comment:
Since you are not adding an annotation to screenContextMock you are not creating a mock for it.
I think in fact that you don't need a mock of ScreenContext, can you try passing null to method requiring such parameters and remove this field?